code/modules/cargo/packs/tools.dm

```diff
@@ -74,6 +74,12 @@
 	contains = list(/obj/item/pickaxe/drill/jackhammer)
 	crate_name = "jackhammer crate"
 
+/datum/supply_pack/tools/jackhammer
```

Comment:
overwrites the other jackhammer crate, probably meant to be tools/plasmacutter or similar.

Reply:
yeeeep. That it is. Thanks for catching that